usaspending_api/etl/es_transaction_template.json

```diff
@@ -142,15 +147,23 @@
           "index": false
         },
         "awarding_toptier_agency_name": {
-          "type": "text",
-          "fielddata": true
+          "type": "keyword",
+          "fields": {
+            "raw": { 
+              "type": "text"
+            }
+          }
         },
         "funding_toptier_agency_name": {
           "type": "text"
         },
         "awarding_subtier_agency_name": {
-          "type": "text",
-          "fielddata": true
+          "type": "keyword",
+          "fields": {
+            "raw": { 
+              "type": "text"
+            }
```

Comment:
This is good in case we want to aggregate via "awarding_subtier_agency_name". But we'll have to append ".raw" to the search fields:

https://www.elastic.co/guide/en/elasticsearch/reference/current/multi-fields.html